pkg/data/management/role_data.go

```diff
@@ -229,7 +229,8 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("catalog.cattle.io").resources("clusterrepos").verbs("get", "list", "watch")
 
 	rb.addRoleTemplate("Manage Cluster Backups", "backups-manage", "cluster", false, false, false).
-		addRule().apiGroups("management.cattle.io").resources("etcdbackups").verbs("*")
+		addRule().apiGroups("management.cattle.io").resources("etcdbackups").verbs("*").
+		addRule().apiGroups("rke.cattle.io").resources("etcdsnapshots").verbs("get", "list", "watch")
```

Comment:
I don't understand why this is get/list and not `*`, but it's also that way on Cluster Owner ? Maybe @Oats87 can explain.

Reply:
We do not necessarily want users being able to delete, update, or patch snapshots manually. These are all supposed to be managed by the controller only, and editing them can lead to unexpected results.